docs/config-definition.md

```diff
@@ -1,9 +1,9 @@
 # Ark Config definition
 
-* [Overview][10]
```

Comment:
I'd prefer to keep the changes to a minimum, and just append the new link. Would you mind doing that @madddi? Sorry for the confusion!